python/golem_task_api/apputils/logging.py

```diff
@@ -0,0 +1,52 @@
+import logging.config
+
+from typing import Dict, List, Optional, Union
+
+# consts
+DEFAULT_LEVEL = logging.INFO
+DEFAULT_EXTERNAL_LOGGERS = ['hpack', 'peewee']
+LOG_LEVEL_VALUES = [
+    'CRITICAL',
+    'ERROR',
+    'INFO',
+    'WARNING',
+    'DEBUG',
+]
+# types
+LogLevelArg = Union[str, int]
+# vars
+logger = logging.getLogger(__name__)
+
+
+def init_logging(
+    log_config: Optional[Dict] = None,
+    log_level_arg: Optional[LogLevelArg] = None,
+    log_level_default: int = DEFAULT_LEVEL,
+    external_loggers: List[str] = DEFAULT_EXTERNAL_LOGGERS,
+    external_log_level: LogLevelArg = DEFAULT_LEVEL
+):
+    level = log_level_arg or log_level_default
+    if log_config:
+        logging.config.dictConfig(log_config)
```

Comment:
I think this function should return after `.dictConfig()` as basically all next lines override what has been defined in `log_config`, thus making it impossible for the app developer to control logging.
Only if `log_config is None` we can continue with next lines.

Reply:
good one, will look into this

Reply:
turned out that by returning after `.dictConfig()` there is nothing left for this function to do.

Removed the argument + call and documented this as a replacement for using `.dictConfig()`